CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing and Acceptance
+
+We're a friendly community, and we would love to have more collaborators!
+
+## Maintenance mode
+
+We view metamath-exe as being in maintenance mode. Although we do envisage
+fixing bugs (at least serious ones) and making small updates to reflect
+ongoing changes to our processes or metamath proof databases (especially
+those in the https://github.com/metamath/set.mm repository), we do not
+encourage developing significant new functionality in metamath-exe.
+
+Why? metamath.exe is the oldest of the metamath tools and is showing its
+age in a lot of ways including that C is probably not the language that
+anyone would choose for this kind of code today.
+
+Here's what Norm Megill, who wrote metamath-exe and did almost all of
+its maintenance until 2021, said:
```

Comment:
```suggestion
its maintenance until his passing in 2021, said:
```
It is helpful to give context on why the maintenance has changed so suddenly.

Reply:
I thought about wording like that and wasn't sure but OK, I'll accept this suggestion. We can always re-revise it later if it somehow the context feels different in a year or two.